TEST_APPS/testcases/netsocket/TCPSERVER_ACCEPT.py

```diff
@@ -94,9 +94,9 @@ def case(self):
         if data != "hello":
             raise TestStepFail("Received data doesn't match the sent data")
 
-        self.command("dut1", "socket " + str(server_socket_id) + " delete")
-        self.command("dut1", "socket " + str(server_base_socket_id) + " delete")
-        self.command("dut2", "socket " + str(self.client_socket_id) + " delete")
+        self.command("dut1", "socket " + str(server_socket_id) + " close")
+        self.command("dut1", "socket " + str(server_base_socket_id) + " close")
+        self.command("dut2", "socket " + str(self.client_socket_id) + " close")
```

Comment:
Could you please move these close-commands to teardown section.
If the test case raise error, then the close-commands are never called